doc/man3/EVP_MAC.pod

```diff
@@ -300,14 +272,25 @@ otherwise B<NULL>.
 
       size_t i;
 
+      OSSL_PARAM params[4];
+      size_t params_n = 0;
+
+      if (cipher != NULL)
+          params[params_n++] =
+              OSSL_PARAM_construct_utf8_string("cipher", cipher,
```

Comment:
Not sure we should be using raw strings everywhere.. would prefer defines..

Reply:
if the name is architecturally defined, that having things like this
```
#define CIPHER_PARAM_NAME "cipher"
```
adds a level of indirection and makes things less readable.

Reply:
I don't mind which we use, but we should at least try to be consistent about it.  Thus far, we've been using defines.  Personally, I'd prefer strings but not enough to argue for them.

One distinct advantage of using defines is that typos are detected at compile time, rather than run time.

Reply:
I don't see the advantage:
```
#define CIPHER_PARAM_NAME "cypher"
```

What am I missing?

Reply:
Its wrong in one place :)

Reply:
I dont care either - it just needs to be consistent. Matt is doing the opposite.

Reply:
This is the user manual.  Leave the choice to the user.